sycl/include/CL/sycl/feature_test.hpp

```diff
@@ -0,0 +1,88 @@
+//==----- feature_test.hpp - SYCL Feature Test Definitions -----*- C++ -*---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------=== //
+
+//
+// IMPORTANT:  feature_test.hpp is a generated file - DO NOT EDIT
```

Comment:
This file got auto generated. Not sure if I did something wrong.

Reply:
Do we need it? Can it be removed?
Not sure that I see a point in committing autogenerated files.

Reply:
Weird

Reply:
I suspect that these are old autogenerated files that you had lying around. These are now generated in `sycl/include/sycl/*` and are in .gitignore as such.